File: src/tool/spring-boot-generator/output/authority/db/systemRoleInfo.js
let systemRoleInfo = {
	name: "systemRoleInfo",
	description: "system role info",
	columns: {
		id: {
			name: "id",
			description: "id",
			isPk: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		systemId: {
			name: "systemId",
			description: "related system",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		role: {
			name: "role",
			description: "role",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		description: {
			name: "description",
			description: "description",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		createUser: {
			name: "createUser",
			description: "",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "",
			nullable: true,
			type: {
				name: "timestamp",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"systemId",
	"role",
	"description",
	"createUser",
	"createTime"
];

function create() {
  return {};
}

systemRoleInfo.columnsArray=columnsArray;
systemRoleInfo.create=create;
module.exports=systemRoleInfo;

File: src/tool/spring-boot-generator/output/authority/authority/src/main/java/com/authority/pojo/entity/SystemRoleInfo.java
package com.authority.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;



@ApiModel("system role info")
@Getter
@Setter
public class SystemRoleInfo  {

    @ApiModelProperty("id")
    private Integer id;

    @ApiModelProperty("related system")
    private Integer systemId;

    @ApiModelProperty("role")
    private String role;

    @ApiModelProperty("description")
    private String description;

    @ApiModelProperty("")
    private String createUser;

    @ApiModelProperty("")
    private Date createTime;

}